task_1.py

```diff
@@ -0,0 +1,27 @@
+# Напишите функцию, которая преобразует pickle файл
+# хранящий список словарей в табличный csv файл.
+# Для тестированию возьмите pickle версию файла из задачи
+# 4 этого семинара.
+# Функция должна извлекать ключи словаря для заголовков
+# столбца из переданного файла.
+
+__all__ = ['my_func']
+
+
+import pickle
+import csv
+
+
+def my_func(file_pickle, file_csv):
+    with (open(file_pickle, 'rb') as f_read,
+          open(file_csv, 'w', newline='', encoding='utf-8') as f_write
+          ):
+        new_dict = pickle.load(f_read)
+        csv_write = csv.DictWriter(f_write, fieldnames=['id', 'level', 'name'],
+                                   dialect='excel-tab', quoting=csv.QUOTE_ALL)
```

Comment:
Как я понял у вас список список словаре, тогда можно сделать так
```suggestion
        csv_write = csv.DictWriter(f_write, fieldnames=new_dict[0],
                                   dialect='excel-tab', quoting=csv.QUOTE_ALL)
```